toolchain/lc-compile/src/main.c

```diff
@@ -102,20 +103,22 @@ usage(int status)
 "Compile a LiveCode Builder source file.\n"
 "\n"
 "Options:\n"
-"      --modulepath PATH    Search PATH for module interface files.\n"
-"      --output OUTFILE     Filename for bytecode output.\n"
-"      --outputc OUTFILE    Filename for C source code output.\n"
-"      --deps make          Generate lci file dependencies in make format for\n"
-"                           the input source files.\n"
-"      --deps order         Generate the order the input source files should be\n"
-"                           compiled in.\n"
-"      --deps changed-order Generate the order the input source files should be\n"
-"                           compiled in, but only if they need recompiling.\n"
-"      --manifest MANIFEST  Filename for generated manifest.\n"
-"      -Werror              Turn all warnings into errors.\n"
-"  -v, --verbose            Output extra debugging information.\n"
-"  -h, --help               Print this message.\n"
-"  --                       Treat all remaining arguments as filenames.\n"
+"      --modulepath PATH      Search PATH for module interface files.\n"
+"      --output OUTFILE       Filename for bytecode output.\n"
+"      --outputc OUTFILE      Filename for C source code output.\n"
+"      --deps make            Generate lci file dependencies in make format for\n"
+"                             the input source files.\n"
+"      --deps order           Generate the order the input source files should be\n"
+"                             compiled in.\n"
+"      --deps changed-order   Generate the order the input source files should be\n"
+"                             compiled in, but only if they need recompiling.\n"
+"      --manifest MANIFEST    Filename for generated manifest.\n"
+"      --interface INTERFACE  Filename for generated interface.\n"
+"      -Werror                Turn all warnings into errors.\n"
+"      --error-info           Emit warnings and errors in form suitable for testing.\n"
```

Comment:
I would still really prefer making the "normal" warning & error format more suitable for automated processing. 😛